packages/exoflex/src/components/RichRadio/RichRadioItem.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+
+import Text from '../Text';
+
+import useTheme from '../../helpers/useTheme';
+
+export type RichRadioItemProps = {
+  label: string;
+  selectedColor?: string;
+  selected?: boolean;
+  uppercase?: boolean;
+  firstItem?: boolean;
+  onPress: () => void;
+  style?: StyleProp<ViewStyle>;
+  textStyle?: StyleProp<TextStyle>;
+  firstItemCustomStyle?: StyleProp<ViewStyle>;
```

Comment:
Usually we don't add the word custom in the style props right?

Reply:
Yeah. My intention was to make the user not confused between `firstItem` and `firstItemStyle`.